tests/lib/mirrors/mirrors_reader.dart

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) 2014, the Dart project authors.  Please see the AUTHORS file
 // for details. All rights reserved. Use of this source code is governed by a
-// BSD-style license that can be found in the LICESNE file.
+// BSD-style license that can be found in the LICENSE file.
```

Comment:
License changes historically spook lawyers, but this shouldn't be a problem...

Reply:
Indeed.